metadata-ingestion/src/datahub/ingestion/source/sql/athena.py

```diff
@@ -383,7 +383,7 @@ def get_database_container_key(self, db_name: str, schema: str) -> DatabaseKey:
         # Based on community feedback, db_name only available if it is explicitly specified in the connection string.
         # If it is not available then we should use schema as db_name
 
-        if not db_name:
+        if not db_name or db_name == "none":
```

Comment:
This seems like a pretty safe change, but I'm a bit confused about when it would be `none` in lowercase

Is this from `str(None).lower()` being `"none"`, or something else?

Reply:
`db_name` is `"none"` when the `database` parameter in the recipe is not set.
To be honest, I haven't checked where this value comes from and why the original if clause does not work.
But I think your assumption sounds reasonable.

This PR is intended as a quick fix, but I'm open to fixing the underlying cause instead.

Reply:
This is the file where I think the above should be changed -> https://github.com/datahub-project/datahub/blob/efaf21d571262a58c0c3e624c523213be4310c43/metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py#L370

Reply:
Sounds good, I'll adapt accordingly

Reply:
Done